Import guides (#42)
fivetran/terraform-provider-fivetran: docs/resources/user.md

- `phone` - The phone number of the user.
- `picture` - The url of the user's avatar.

> - `role` - The account role that you would like to assign this new user to. Possible values: Account Administrator, Account Billing, Account Analyst, Account Reviewer, Destination Creator, or a custom role with account-level permissions. You can find all roles at your (Roles)[https://fivetran.com/account/roles] page at Fivetran Dashboard.

Reviewer: ```suggestion
- `role` - The account role that you would like to assign this new user to. Possible values: Account Administrator, Account Billing, Account Analyst, Account Reviewer, Destination Creator, or a custom role with account-level permissions. You can find available roles on the [Roles tab](https://fivetran.com/account/roles) tab of the account management page in your Fivetran dashboard.
```

- `last_updated` 
- `logged_in_at` 
- `verified` 

## Import

> To import an existing `fivetran_user` resource into your terraform state you need to get `user_id`. 

Reviewer: ```suggestion
1. To import an existing `fivetran_user` resource into your Terraform state, you need to get `user_id`. 
```

> You can retrieve all users via [Data Source: fivetran_users](/docs/data-sources/users)

Reviewer: ```suggestion
You can retrieve all users using the [fivetran_users data source](/docs/data-sources/users).
```

> Then define an empty resource in your .tf configuration:

Reviewer: ```suggestion
2. Define an empty resource in your `.tf` configuration:
```

```hcl
resource "fivetran_user" "my_imported_fivetran_user" {

}
```

> And call `terraform import` command:

Reviewer: ```suggestion
3. Call the `terraform import` command:
```

```
terraform import fivetran_user.my_imported_fivetran_user <user_id>
```

> Then copy-paste values from the state to your .tf config, use `terraform state show`:

Reviewer: ```suggestion
4. Copy and paste values from the state to your `.tf` configuration using the `terraform state show` command:
```

> 4. Copy and paste values from the state to your `.tf` configuration using the `terraform state show` command:

Reviewer: ```suggestion
4. Use the `terraform state show` command to get the values from the state:
```

- `last_updated` 
- `logged_in_at` 
- `verified` 

## Import

1. To import an existing `fivetran_user` resource into your Terraform state, you need to get `user_id`. 
You can retrieve all users using the [fivetran_users data source](/docs/data-sources/users).

2. Define an empty resource in your `.tf` configuration:

```hcl
resource "fivetran_user" "my_imported_fivetran_user" {

}
```

> 3. Call the `terraform import` command:

Reviewer: ```suggestion
3. Run the `terraform import` command:
```

```
terraform import fivetran_user.my_imported_fivetran_user <user_id>
```

4. Use the `terraform state show` command to get the values from the state:

```
terraform state show 'fivetran_user.my_imported_fivetran_user'
```

> 5. copy the values and paste them to your `.tf` configuration.

Reviewer: ```suggestion
5. Copy the values and paste them to your `.tf` configuration.
```

> 5. Paste the connector properties from the state to your `.tf` configuration.

Reviewer: ```suggestion
5. copy the values and paste them to your `.tf` configuration.
```
